TKGame/Components/Concrete/Player_PhysicsComponent.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.Components.Interface;
+
+namespace TKGame.Components.Concrete
+{
+    internal class Player_PhysicsComponent : PhysicsComponent
+    {
+        private static readonly float GRAVITY = 1.0f;
+        void PhysicsComponent.Update(Entity /*&*/entity, GameTime gameTime/*, World &world*/) // The &reference isn't working.
```

Comment:
hope this helps 👍 https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/ref

Reply:
Yeah, I'm a silly guy who forgot to just look it up